config/default_configs/default_config.yml

```diff
@@ -99,6 +99,9 @@ dt_rad:
 idealized_clouds:
   help: "Use idealized clouds in radiation model [`false` (default), `true`]"
   value: false
+prescribe_clouds_in_radiation:
+  help: "Use prescribed clouds in radiation model. Clouds are read from ERA5 data and updated every time radiation is called. The year 2010 is used and continuously repeated. This mode only affect radiation and is only relevant for the RRTGMP mode. [`false` (default), `true`]"
```

Comment:
@szy21 I think there is a typo here - above in `NEWS.md` it says monthly data from 2020 is used whereas here the 2010 dataset is indicated.

Reply:
Thanks. It's a typo in the NEWs. I'll update it.